webapp/channels/src/components/admin_console/access_control/test_modal/test_modal.tsx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect} from 'react';
+import {Modal} from 'react-bootstrap';
+import {FormattedMessage} from 'react-intl';
+
+import type {AccessControlTestResult} from '@mattermost/types/admin';
+
+import SearchableUserList from 'components/searchable_user_list/searchable_user_list_container';
+
+import type {ModalData} from 'types/actions';
+
+import './test_modal.scss';
+
+const USERS_PER_PAGE = 10;
+
+type Props = {
+    testResults: AccessControlTestResult | null;
+    onExited: () => void;
+    onLoad?: () => void;
+    actions: {
+        openModal: <P>(modalData: ModalData<P>) => void;
+        setModalSearchTerm: (term: string) => void;
+    };
+}
+
+const TestResultsModal: React.FC<Props> = ({testResults, onExited, onLoad, actions}) => {
+    useEffect(() => {
+        onLoad?.();
```

Comment:
The `onLoad` prop is not being passed anywhere right now. If its still needed for future, currently it will be called every time onLoad changes, and not just once when component mounts. Not sure whats the intended behaviour, but if needs to run just once, we need to track it in a useRef or something that it has called onLoad once and never call it again.